django/template/loader_tags.py

```diff
@@ -207,6 +210,11 @@ def render(self, context):
         except Exception:
             if context.template.engine.debug:
                 raise
+            else:
+                template_name = getattr(context, 'template_name', 'unknown')
+                logger.warn('template - Exception raised while rendering {{% include %}} '
```

Comment:
to be honest, neither do I. I thought I picked it up from a logger elsewhere in the code base but now I can't find what I thought I was referencing so I'm going to remove it.